docs/UserGuide.md

```diff
@@ -113,23 +113,25 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
-### Locating persons by name: `find`
+### Search students by name or ID: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds students whose names contain any of the given keywords or whose ID matches givens student ID
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Formats: 
+* search by name: `find n/KEYWORD [MORE_KEYWORDS]`
```

Comment:
Would n/NAME be more suitable ?

Reply:
accepted suggection to change from n/KEYWORD to find n/NAME